src/components/DefaultManifestTable/Table.js

```diff
@@ -31,29 +35,53 @@ Headers.propTypes = {
   columnIndexs: PropTypes.array.isRequired
 }
 
-const Rows = ({ rowCount, columnIndexs }) => {
+const Rows = ({ rowCount, columnIndexs, trPropsHandler, tdPropsHandler }) => {
   const rowIndexs = useMemo(() => integerSequence(rowCount), [rowCount])
 
   return (
     <tbody>
-      {rowIndexs.map(i => <Row key={i} rowIndex={i} columnIndexs={columnIndexs} />)}
+      {rowIndexs.map(i => <TableRow key={i} rowIndex={i} columnIndexs={columnIndexs} trPropsHandler={trPropsHandler} tdPropsHandler={tdPropsHandler} />)}
     </tbody>
   )
 }
 
 Rows.propTypes = {
   rowCount: PropTypes.number.isRequired,
-  columnIndexs: PropTypes.array.isRequired
+  columnIndexs: PropTypes.array.isRequired,
```

Comment:
Indexes or Indices

Reply:
Agree.

Reply:
Indexes, that was bothering me last night but didn't want to change it in this PR thinking it would cause problems.  Now I am looking at it closer I think it is fine. Will change it now.